tools/distrib/sanitize.sh

```diff
@@ -20,7 +20,7 @@ cd $(dirname $0)/../..
 DIFF_COMMAND="git diff --name-only HEAD | grep -v ^third_party/"
 
 if [ "x$1" == 'x--pre-commit' ]; then
-  if eval $DIFF_COMMAND | grep '^build.yaml$'; then
+  if eval $DIFF_COMMAND | grep '^build_handwritten.yaml$'; then
```

Comment:
Should `BUILD` be checked here? because `BUILD` and `build_handwritten.yaml` are now sources of truth.

Reply:
Good point but actually adding BUILD in repo root won't be enough as the bazel metatadata also comes from other BUILD files as well as some .bzl files. I think the best option is to nuke the `--pre-commit` flag, I doubt anyone actually uses it (and I found any other occurrences of it in our repo).

Reply:
Removed --pre-commit option